src/components/Home.js

```diff
@@ -5,19 +5,44 @@ import styled from 'styled-components/native'
 
 import LeftNav from './LeftNav'
 import MainArea from './MainArea'
+import InitialState from './InitialState'
+
+type State = {
+  initial: boolean,
+}
 
 const Root = styled.View`
   width: 100vw;
   height: 100vh;
   flex: 1;
   flex-direction: row;
 `
-class Home extends Component<{}> {
+class Home extends Component<{}, State> {
+  state = {
+    initial: true,
+  }
+
+  componentDidMount() {
+    // there will likely be a better/ additional deciding factor here
+    // once integrated w/ swarm
+    if (localStorage.getItem('local-storage-session-key')) {
```

Comment:
What is the purpose of this key please?

Reply:
I see this as a temporary solution for generating a session key to access the local storage data. I thought it might be too transparent to have the array stored in some string location, although I guess it's not really doing too much to prevent that, it's just one extra step. I can just store the array directly for now since this is probably something that'll change with swarm integration I assume (i.e. access local storage via hash of ur public key?)